darts/tests/models/forecasting/test_chronos2.py

```diff
@@ -0,0 +1,460 @@
+from pathlib import Path
+from unittest.mock import patch
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from darts import TimeSeries, concatenate
+from darts.datasets import ElectricityConsumptionZurichDataset
+from darts.tests.conftest import TORCH_AVAILABLE, tfm_kwargs
+from darts.utils.likelihood_models import GaussianLikelihood
+from darts.utils.timeseries_generation import (
+    gaussian_timeseries,
+    linear_timeseries,
+    sine_timeseries,
+)
+
+if not TORCH_AVAILABLE:
```

Comment:
we need to skip these tests before importing any torch-related files (e.g. GaussianLikelihood). These no-torch flavors will only be tested once the PR was merged.

The easiest is to just skip before importing any darts modules.